docs/source/gbd2017_models/concept_models/vivarium_sanofi_zenon/concept_model.rst

```diff
@@ -123,12 +123,12 @@ The probability of side effects is also taken from the literature, and if a pers
 BAU parameter data tables
 +++++++++++++++++++++++++
 
-.. csv-table:: Current prescriptions
-   :file: current_rx.csv
-   :widths: 30, 20, 10, 10
+.. csv-table:: Table 1: Probability of having LDL-c measured
+   :file: prob_testing_ldlc.csv
+   :widths: 20, 10, 10
    :header-rows: 1
 
-.. csv-table:: Probability of prescription given high LDL-c
+.. csv-table:: Table 2: Probability of Rx given high LDL-c
    :file: prob_rx_given_high_ldlc.csv
    :widths: 20, 10, 10
```

Comment:
At a follow-up visit, if a patient has not reached their target (defined as 50% reduction in their untreated LDL-C), they may be given a higher dose, a 2nd drug or a different statin (if on statin).  The probability of each is given in Table 5.  These numbers are global, not location specific.